library/kong

```diff
@@ -2,21 +2,21 @@ Maintainers: Kong Docker Maintainers <support@konghq.com> (@thekonginc)
 GitRepo: https://github.com/Kong/docker-kong.git
 # needs "Constraints" to match "centos" (which this image is "FROM")
 
-Tags: 1.4.0rc2-alpine, 1.4.0rc2, 1.4rc2
-GitCommit: 51ca0adb412814c84accc80b4fbc76b88b36dfe5
-GitFetch: refs/tags/1.4.0rc2
+Tags: 1.4.0-alpine, 1.4.0, 1.4
```

Comment:
After I merge, I realized that you may want to move the plain `alpine`, `latest`, `ubuntu`, and `centos` tags up from the `1.3` images?

Reply:
You are absolutely right, thanks for pointing this out.

I've made such change in #6855 

Apologies if this produces any inconvenience.